compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h

```diff
@@ -51,6 +51,7 @@ constexpr StringLiteral kTuningSpecEntrypointAttrName =
 constexpr StringLiteral kSerializedTuningSpecAttrName =
     "iree_codegen.tuning_spec_mlirbc";
 constexpr StringLiteral kKernelConfigSpecName = "__kernel_config";
+static constexpr StringLiteral kUKernelDescriptorName = "iree_codegen.ukernel";
```

Comment:
```suggestion
constexpr StringLiteral kUKernelDescriptorName = "iree_codegen.ukernel";
```

Reply:
I moved it back to the cpp file as I am using the getter now.